reference/halyard/component-sizing.md

```diff
@@ -38,15 +43,31 @@ deploymentEnvironment:
       requests:
         cpu: 100m
         memory: 128Mi
+    # 2. This applies sizings to the clouddriver container as well as any sidecar 
+    # containers running with clouddriver.
+    spin-clouddriver:
+      limits:
+        cpu: 1
+        memory: 1Gi
+      requests:
+        cpu: 250m
+        memory: 512Mi
 ```
 
 Limits and requests follow the Kubernetes conventions [documented here](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/).
 
 #### Updating JAVA_OPTS
 
-The shaping of the pods via request and limits will not cap the java processes for the Spinnaker microservice. This can be achieved by using the `env` key in [service-settings](/reference/halyard/custom/#tweakable-service-settings).
+As of Halyard 1.8, all JVM-based services will automatically have the following JAVA_OPTS set:
+
+```
+JAVA_OPTS=-XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap -XX:MaxRAMFraction=2
+```
+
+This sets the JVMs heap to half the memory allocated per container. This can be overriden by specifying your own `JAVA_OPTS` 
+using the `env` key in [service-settings](/reference/halyard/custom/#tweakable-service-settings).
 
-In general, the `-Xms` will follow 80%-90% of the requests memory allotment and `-Xmx` will follow 80-90% of the limits memory allotment. For the clouddriver example above, the `env` key is as follows:
+In general, the `-Xms` should be 80%-90% of the requests memory allotment and `-Xmx` should be 80-90% of the limits memory allotment. For the clouddriver example above, the `env` key is as follows:
```

Comment:
Can we be more specific about the "in general" part? And "should"?

Like:
"Set -Xms to 80-90% of the requests memory allotment, unless..."

Reply:
Ideally yes, but we simply don't know/have the experience.